### PR TITLE
crun: update to 1.8.5.

### PR DESCRIPTION
**Pull request by daniel-eys**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/crun/template [hunk 1]
```diff
@@ -25,13 +27,6 @@ case "$XBPS_MACHINE" in
...
 
-do_check() {
-	if [ "$XBPS_CHECK_PKGS" = full ]; then
-		# most of the tests don't work inside a container
-		make check ${makejobs}
-	fi
-}
-
 post_install() {
 	rm -f ${DESTDIR}/usr/lib/*.a
```
Comment (paper42) [inline srcpkgs/crun/template:31]:
```suggestion
	rm ${DESTDIR}/usr/lib/*.a
```

File: srcpkgs/crun/template [hunk 2]
```diff
@@ -25,13 +27,6 @@ case "$XBPS_MACHINE" in
 	*) ;;
 esac
```
Comment (paper42) [inline srcpkgs/crun/template:28]: I don't think this is necessary, we are only matching architecture on the host machine, mips* is missing and we will just forget to add more archs when we add them.

Comment (daniel-eys) [reply srcpkgs/crun/template:28]: I guess this was added for the same reason as this: https://github.com/void-linux/void-packages/commit/61ef978
Maybe @q66 can comment here

File: srcpkgs/crun/template (hunk 1 above)
Comment (daniel-eys) [reply srcpkgs/crun/template]: done

**Pull request merged**